docs/DeveloperGuide.adoc

```diff
@@ -279,13 +279,13 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* has a need to manage a significant number of delivery tasks and drivers
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: manage delivery tasks faster than a typical mouse/GUI driven app
 
 [appendix]
 == User Stories
```

Comment:
1. Missing the user stories in which the user wants to create, update, or delete deliveries.
2. Perhaps a nice user story would be to prompt the user when there are insufficient drivers to accommodate all deliveries within a certain timeframe.